### PR TITLE
Add Support for WP Uninstall

### PR DESCRIPTION
**Pull request by kjschulz**
~~@josegonzalez, this doesn't execute properly just yet.  Thought I was using the same syntax you did but I'm missing something.  Mind to take a look when you get a chance?  Upon execution I get the following error and I'm not sure what I'm doing wrong: `make: *** No rule to make target 'destroy'.  Stop.`~~

### CODE REVIEW

**Comment by kjschulz** [comment]
~~Also @josegonzalez, just tried these commands manually and it seems that the `-force` flag for `apps:destroy` isn't recognized (i'm on dokku 7.2).  And it would seem that if we do destroy the app, unmounting of the storage dirs is unnecessary...we only need to `rm` them on the server.  You agree?  If so I'll make that change and push as well...~~

**Comment by josegonzalez** [comment]
It's two dashes, not one.

**Comment by kjschulz** [comment]
Ok, should be all good to go now 🎉

File: Makefile [hunk 1]
```diff
@@ -116,3 +116,25 @@ else
...
+.PHONY: destroy
+destroy: ## destroys an existing wordpress blog installation and outputs undeploy instructions
+ifndef APP_NAME
+	$(error "Missing APP_NAME environment variable, this should be the name of your blog app")
+endif
+ifndef SERVER_NAME
+	$(error "Missing SERVER_NAME environment variable, this should be something like 'dokku.me'")
+endif
+	# destroy the mysql database
+	$(DOKKU_CMD) mysql:unlink $(APP_NAME)-database $(APP_NAME)
```
Comment (josegonzalez) [inline Makefile:148]: This  assumes unattended. Can you also create the alternative when its not unattended?

**Review by josegonzalez — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (josegonzalez) [reply Makefile:148]: Specifically, we should have a version that echos out the commands to run.

**Comment by kjschulz** [comment]
Yep, will try and do this tonight or tomorrow.  Thanks.

File: Makefile (hunk 1 above)
Comment (kjschulz) [reply Makefile:148]: Yep, will try and do this tonight/tomorrow.

Comment (kjschulz) [reply Makefile:148]: @josegonzalez, made the change.

**Review by josegonzalez — APPROVED** [review]

**Pull request merged**